### ISSUE TITLE
exports['tearDown'] usage with SocketIO server testing

### ISSUE THREAD

**Issue by perezd** [cloudkick/whiskey#3]
I want to be able to write the following code:

``` CoffeeScript
exports['setUp'] = (cb) ->
  SocketServer.control.startServer(cb)

exports['tearDown'] = (cb) ->
  SocketServer.control.startServer(cb)
```

Where setUp happens before all tests run, and tearDown happens after all the tests have run. Ideally, we are only running the server startup once, so all tests can async happen against the same server. Let me know if I can provide more help on this issue.

**Comment by Kami** [comment]
This issue has been addressed in Whiskey 0.3.0.

Keep in mind that tearDown is not guaranteed to run if a timeout is reached.

**Comment by perezd** [comment]
Working great, thanks!!!
